airbyte-webapp/src/locales/en.json

```
@@ -91,7 +91,7 @@
   "form.dockerError": "Could not find docker image",
   "form.edit": "Edit",
   "form.done": "Done",
-  "form.examples": "(e.g. {examples})",
+  "form.examples": "( e.g. {examples} )",
```

Would it potentially be better to just remove the parentheses completely, if that avoids bad copy-pastes entirely?

---

That's possible. What about
```suggestion
  "form.examples": "E.g. {examples}"
```

The minor issue is that there is no closing full stop at the end. So technically it is wrong if we ask a high school grammar teacher. But there is no full stop currently already. So I think that can be ignored.

---

Yeah I think this should be fine!